spotlight-client/src/contentModels/HistoricalPopulationBreakdownMetric.ts

```diff
@@ -196,4 +199,44 @@ export default class HistoricalPopulationBreakdownMetric extends Metric<Historic
 
     return countsByDate.length ? countsByDate : undefined;
   }
+
+  get readme(): string {
```

Comment:
the formatting logic previously used by the UI is already generic for different types of unknowns; you ought to be able to port all of that into this function. Then it will no longer be specific to this metric type and you can lift it up to the Metric superclass

Reply:
Now the Metric superclass should handle generating the readme